src/euphorie/deployment/profiles/default/registry.xml

```diff
@@ -311,4 +311,13 @@
     </field>
     <value purge="false">--margin-bottom 1cm --margin-left 1.5cm --margin-right 1.5cm --margin-top 2cm --disable-javascript --viewport-size 10000x10000</value>
   </record>
+  <record name="plone.enable_self_reg" interface="Products.CMFPlone.interfaces.controlpanel.ISecuritySchema" field="enable_self_reg">
+    <field type="plone.registry.field.Bool">
```

Comment:
The field element can be omitted if I am not wrong, given that Plone should already have it.